### DISCUSSION TITLE
changing monitor display output type

### DISCUSSION THREAD

**Discussion by yarintm** [MonitorControl/MonitorControl#827] [Ideas]
Hi I have an idea for a new feature, adding the ability to change the monitor source(i.e from HDMI 1 to HDMI 2).
I will be happy to work on this as well :)

**Comment by waydabber** [comment]
hi, yes, this is possible and is a planned feature: https://github.com/MonitorControl/MonitorControl/issues/720

you can of course work on this if you'd like and create a pull request. If the solution fits the app and is approved by the devs, it will be added to the app!
